src/TurbulenceClosures/turbulence_closure_implementations/smagorinsky_lilly.jl

```diff
@@ -16,7 +16,7 @@ struct SmagorinskyLilly{TD, FT, P} <: AbstractScalarDiffusivity{TD, ThreeDimensi
 end
 
 @inline viscosity(::SmagorinskyLilly, K) = K.νₑ
-@inline diffusivity(::SmagorinskyLilly, K, ::Val{id}) where id = K.νₑ / closure.Pr[id]
+@inline diffusivity(closure::SmagorinskyLilly, K, ::Val{id}) where id = K.νₑ / closure.Pr[id]
```

Comment:
This failure would be caught just by calling `diffusivity(model.closure, model.diffusivity_fields, Val(:c))` right?

Reply:
Right!